src/components/common/Balance.test.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { shallow, mount, configure, render } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+
+import Balance from './Balance'
+import milli from '../../images/n-1000.svg'
+
+configure({adapter: new Adapter()});
+
+describe('<Balance.js>', ()=>{
+    const contextSmall = "1"+"0".repeat(10) 
+    const contextBig = "1"+"0".repeat(21)
+
+    it('balance should return properly for small number',()=>{
+        let wrapper = shallow(<Balance amount={contextSmall} />)
+        expect(wrapper.contains("0.00001")).toEqual(true);
```

Comment:
you might also want to match DOM against snapshot https://jestjs.io/docs/en/snapshot-testing

Reply:
thank you for telling me this. I think it might be more useful for bigger component. Next time, when the component needs to check many places, I will use this. :)